### PR TITLE
feat(Funnels): show number of users that entered/completed funnel

### PR DESCRIPTION
**Pull request by timgl**
## Problem

The calculation of funnel trends is different to normal funnels. Users can enter the funnel multiple times, whereas in a normal funnel they can only do this once. This means percentages can be very different. This change at least will make it easier for users to understand why this happens

See [this support ticket (internal)](https://posthoghelp.zendesk.com/agent/tickets/44)

## Changes

On hover, display the # of users that entered and completed funnel 
image

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

unit tests + manual QA:
- create funnel trends and hover

### CODE REVIEW

File: frontend/src/scenes/funnels/FunnelLineGraph.tsx [hunk 1]
```diff
@@ -30,20 +29,26 @@ export function FunnelLineGraph({
...
-                renderSeries: (_, datum) => {
-                    if (!steps?.[0]?.days) {
-                        return 'Trend'
-                    }
+                renderSeries: function renderSeries() {
                     return (
-                        getFormattedDate(steps[0].days?.[datum.dataIndex], filters.interval) +
-                        ' ' +
-                        shortTimeZone(insight.timezone)
+                        <div style={{ paddingTop: 20, height: 68 }}>
```
Comment (timgl) [inline frontend/src/scenes/funnels/FunnelLineGraph.tsx:36]: this is quite hacky but couldn't find a better way

File: frontend/src/scenes/insights/InsightTooltip/insightTooltipUtils.tsx [hunk 2]
```diff
@@ -34,7 +36,7 @@ export interface TooltipConfig {
     rowCutoff?: number
     colCutoff?: number
     renderSeries?: (value: React.ReactNode, seriesDatum: SeriesDatum, idx: number) => React.ReactNode
-    renderCount?: (value: number, seriesDatum: SeriesDatum | InvertedSeriesDatum, idx: number) => React.ReactNode
```
Comment (alexkim205) [inline frontend/src/scenes/insights/InsightTooltip/insightTooltipUtils.tsx:37]: What's the reasoning for removing this type?

File: frontend/src/scenes/funnels/FunnelLineGraph.tsx [hunk 3]
```diff
@@ -30,20 +29,26 @@ export function FunnelLineGraph({
...
+                            Entered/completed
+                        </div>
                     )
                 },
-                renderCount: (count) => {
-                    return `${count}%`
+                renderCount: function renderCount(count, datum) {
+                    if (!datum.reached_from_step_count || !datum.reached_to_step_count) {
+                        return null
+                    }
```
Comment (alexkim205) [inline frontend/src/scenes/funnels/FunnelLineGraph.tsx:46]: This introduces a bug where series labels are displayed but counts are omitted where percentage is 0.

Screen Shot 2022-06-09 at 1 37 21 PM

**Review by alexkim205 — CHANGES_REQUESTED** [review]
left some inline comments

**Comment by alexkim205** [comment]
@timgl I'm going to remove the stale label since you were out last week

**Pull request closed without merging**